### PR TITLE
feat: Add log configuration for CA, node and chaincode

### PR DESCRIPTION
**Pull request by vponselvan**
Added option to configure the logs for CA, node and chaincode. Logs are enabled by default and have to be disabled explicitly if not required. Also, removed the feature from the list on README.

### CODE REVIEW

File: src/network.ts [hunk 1]
```diff
@@ -309,7 +321,7 @@ export class HyperledgerFabricNetwork extends constructs.Construct {
     // can't populate their outputs fully until later, which is annoying
     const nodeIds = this.nodes.map(n => n.nodeId);
     const nodeArns = nodeIds.map(i => `arn:${partition}:managedblockchain:${region}:${account}:nodes/${i}`);
-    const dataSdkCallPolicy = customresources.AwsCustomResourcePolicy.fromSdkCalls({
+    const sdkCallPolicy = customresources.AwsCustomResourcePolicy.fromSdkCalls({
```
Comment (lordjabez) [inline src/network.ts:324]: Nice little rename here.

File: src/network.ts [hunk 2]
```diff
@@ -100,6 +100,12 @@ export interface HyperledgerFabricNetworkProps {
    */
   readonly thresholdComparator?: ThresholdComparator;
 
+  /**
+   * The configuration to enable or disable certificate authority logging
+   * @default - true
+   */
+  readonly loggingConfiguration?: boolean;
```
Comment (lordjabez) [inline src/network.ts:107]: Let's make this more explicit, how about `enableCaLogging` ?

File: src/node.ts [hunk 3]
```diff
@@ -56,6 +56,12 @@ export interface HyperledgerFabricNodeProps {
    */
   readonly instanceType?: InstanceType;
 
+  /**
+   * The configuration to enable or disable node and chaincode logging
+   * @default - true
+   */
+  readonly loggingConfiguration?: boolean;
```
Comment (lordjabez) [inline src/node.ts:63]: Let's split this out into two:
`enableNodeLogging` and `enableChaincodeLogging`

**Review by lordjabez — CHANGES_REQUESTED** [review]
I don't think there is a need for changes in package-lock.json or yarn.lock to support these changes. Can you remove those files from the commit?

**Comment by vponselvan** [comment]
Updated following:
* Separated logging for node and chaincode
* Renamed log variables
* Removed package-lock.json and yarn.lock from the PR

File: src/network.ts [hunk 4]
```diff
@@ -334,25 +346,49 @@ export class HyperledgerFabricNetwork extends constructs.Construct {
...
     this.ordererEndpoint = networkData.getResponseField('Network.FrameworkAttributes.Fabric.OrderingServiceEndpoint');
     this.caEndpoint = memberData.getResponseField('Member.FrameworkAttributes.Fabric.CaEndpoint');
 
     // As stated earlier, node constructs can't populate all their properties
     // until after the above network and member SDK calls succeed; thus the
-    // function call below where the fetches are split out
-    for (const n of this.nodes) n.fetchData(dataSdkCallPolicy);
+    // function calls below where the fetches are split out and logging is configured
+    for (const n of this.nodes) {
+      n.configureLog(sdkCallPolicy);
```
Comment (lordjabez) [inline src/network.ts:389]: One more minor tweak: let's call this function `configureLogging`

**Review by lordjabez — COMMENTED** [review]
Can you update with my one more suggestion, and also please squash all your commits into one (and run a final eslint check), then we can merge.

**Comment by vponselvan** [comment]
Updated the function name and squashed my commits into one.

**Review by lordjabez — APPROVED** [review]

**Pull request merged**